src/routes/user-role-settings.route.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import authMiddleware from '../middlewares/auth.middleware';
+import UserRoleSettingsController from '../controllers/user-role-settings.controller';
+import validateUserRole from '../validations/user-role-settings.validation';
+
+const router = express.Router();
+
+router.patch('/reset-roles', authMiddleware.checkUserLoggedIn, validateUserRole, authMiddleware.checkUserRole, UserRoleSettingsController.changeUserRole);
```

Comment:
@adesege  here i have changed from `reset-role` to `reset-roles` as you required.